models/topic.model.js

```diff
@@ -62,5 +61,20 @@ const addArticleComment = (article_id, username, body) => {
       return rows[0];
     });
 };
+const updateArticleVotes = (article_id, inc_votes) => {
+  return db.query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject(new Error("Not Found"));
```

Comment:
When return a Promise.reject. it would be better to send back an object instead a new Error
e.g.`return Promise.reject({status:404, msg: "article not found"})`

Reply:
This piece of logic an be on line75.5, or a previous model fetchArticleId can be reused instead of writing repeated code